module-4/backend/week_two.md

```diff
@@ -5,18 +5,43 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's one difference between ES5 and ES6?
+arrow functions
+
 2. What's the difference between asynchronous and synchronous JavaScript? 
+asynchronous can run multiple things at the same time totally independant of each other
+
 3. What are the four pillars of Object Oriented programming?
+Abstraction, Encapsulation, Inheritance, Polymorphism
+
 4. What are some tools available in JavaScript to help you write object oriented code?
+classes, objects, constructor functions, etc.
+
 5. What are some key concepts to be aware of when refactoring your JavaScript?
+Be consistant, only use ES6 or ES5.  Seperate by functionaliy, like AJAX calls, etc.
+
 6. What's a callback function and what are some reasons when we use/need callback functions?
+A callback function, also known as a higher-order function, is a function that is passed to another function as a parameter, and the callback function is called (or executed) inside the function. These 'first class functions' are unique to JS and are used often, on prettymuch every JQuery method. 
+
 7. What's the scope of variables in Javascript?
+JavaScript has two scopes – global and local. Any variable declared outside of a function belongs to the global scope, and is therefore accessible from anywhere in your code. Each function has its own scope, and any variable declared within that function is only accessible from that function and any nested functions. Because local scope in JavaScript is created by functions, it’s also called function scope. When we put a function inside another function, then we create nested scope.
+
 8. What's the difference between `==` and `===` in JavaScript?
+If you don't know the difference than always use '==='!  "===" is more strict and usually the one a programmer would use. 
```

Comment:
what is the difference though? what type of equality check do these operators make?